config/locales/gobierto_dashboards/views/es.yml

```diff
@@ -13,6 +13,7 @@ es:
 
       Luego podrás configurarla."
     objective: Objetivo
+    ole: 1
```

Comment:
Careful

Reply:
It means: **O**bject **l**inted **e**xactly = 1 (true) :trollface: